include/ConfigManager.h

```diff
@@ -93,184 +114,165 @@ class EXPORT ConfigManager
 	bool hasWorkingDir() const;
 
-	void addRecentlyOpenedProject( const QString & _file );
+	void addRecentlyOpenedProject(const QString & _file);
 
-	const QString & value( const QString & cls,
-					const QString & attribute ) const;
-	const QString & value( const QString & cls,
+	const QString & value(const QString & cls,
+					const QString & attribute) const;
+	const QString & value(const QString & cls,
 					const QString & attribute,
-					const QString & defaultVal ) const;
-	void setValue( const QString & cls, const QString & attribute,
-						const QString & value );
-	void deleteValue( const QString & cls, const QString & attribute);
+					const QString & defaultVal) const;
+	void setValue(const QString & cls, const QString & attribute,
+						const QString & value);
+	void deleteValue(const QString & cls, const QString & attribute);
 
-	void loadConfigFile( const QString & configFile = "" );
+	void loadConfigFile(const QString & configFile = "");
 	void saveConfigFile();
 
 
-	void setWorkingDir( const QString & _wd );
-	void setVSTDir( const QString & _vd );
-	void setArtworkDir( const QString & _ad );
-	void setLADSPADir( const QString & _fd );
-	void setVersion( const QString & _cv );
-	void setSTKDir( const QString & _fd );
-	void setDefaultSoundfont( const QString & _sf );
-	void setBackgroundArtwork( const QString & _ba );
-	void setGIGDir( const QString & gd );
-	void setSF2Dir( const QString & sfd );
+	void setWorkingDir(const QString & workingDir);
+	void setVSTDir(const QString & vstDir);
+	void setLADSPADir(const QString & ladspaDir);
+	void setSF2Dir(const QString & sf2Dir);
+	void setSF2File(const QString & sf2File);
+	void setSTKDir(const QString & stkDir);
+	void setGIGDir(const QString & gigDir);
+	void setThemeDir(const QString & themeDir);
+	void setBackgroundPicFile(const QString & backgroundPicFile);
+	//void setVersion(const QString & _cv); <- This doesn't seem to be used anywhere.
 
-	// creates the working directory & subdirectories on disk.
+	// Creates the working directory & subdirectories on disk.
 	void createWorkingDir();
 
 
 private:
 	static ConfigManager * s_instanceOfMe;
 
 	ConfigManager();
-	ConfigManager( const ConfigManager & _c );
+	ConfigManager(const ConfigManager & _c);
 	~ConfigManager();
 
 	void upgrade_1_1_90();
 	void upgrade_1_1_91();
 	void upgrade();
 
-	QString m_lmmsRcFile;
 	QString m_workingDir;
 	QString m_dataDir;
-	QString m_artworkDir;
 	QString m_vstDir;
-	QString m_ladDir;
-	QString m_gigDir;
+	QString m_ladspaDir;
 	QString m_sf2Dir;
-	QString m_version;
+#ifdef LMMS_HAVE_FLUIDSYNTH
+	QString m_sf2File;
+#endif
 #ifdef LMMS_HAVE_STK
 	QString m_stkDir;
 #endif
-#ifdef LMMS_HAVE_FLUIDSYNTH
-	QString m_defaultSoundfont;
-#endif
-	QString m_backgroundArtwork;
+	QString m_gigDir;
+	QString m_themeDir;
+	QString m_backgroundPicFile;
+	QString m_lmmsRcFile;
 	QStringList m_recentlyOpenedProjects;
+	QString m_version;
 
```

Comment:
In the configmanager. Cpp in the constructor the order of initialization of this members is different than the declaration here. I guess therefore Travis check failed.

Reply:
Fixed the problem with https://github.com/LMMS/lmms/pull/3820/commits/edac2edf01b7a9f2d68f8292c77784c5ef811fee.
Thanks! 👍